include/effects/matrix/PatternWeather.h

```diff
@@ -114,7 +113,7 @@ class PatternWeather : public LEDStripEffect
     {
         return K - 273.15;
     }
-    
+

```

Comment:
This reads backwards to me?  If UseCelcius is true, shouldn't it do K->C instead of K->F?